disposebag/src/main/kotlin/com/kizitonwose/android/disposebag/DisposeBag.kt

```diff
@@ -44,22 +38,28 @@ class DisposeBag @JvmOverloads constructor(owner: LifecycleOwner,
         composite.dispose()
     }
 
-    override fun add(d: Disposable) = composite.add(d)
+    override fun add(disposable: Disposable) = composite.add(disposable)
 
-    override fun remove(d: Disposable) = composite.remove(d)
+    override fun remove(disposable: Disposable) = composite.remove(disposable)
 
-    override fun delete(d: Disposable) = composite.delete(d)
+    override fun delete(disposable: Disposable) = composite.delete(disposable)
 
     override fun onPause(owner: LifecycleOwner) {
-        if (event == Lifecycle.Event.ON_PAUSE) dispose()
+        if (event == Lifecycle.Event.ON_PAUSE) {
+            dispose()
+        }
     }
 
     override fun onStop(owner: LifecycleOwner) {
-        if (event == Lifecycle.Event.ON_STOP) dispose()
+        if (event == Lifecycle.Event.ON_STOP) {
+            dispose()
```

Comment:
The code follows the [official Kotlin Style guide for Android](https://android.github.io/kotlin-guides/style.html#braces), hence braces are not really needed in this case.

Reply:
Yeah this is subjective